File: beginner/1017/fuel_spent.c
// challenge link - https://judge.beecrowd.com/en/problems/view/1017

#include <stdio.h>

int main(void) {
	int traveltime; // hours
	int avgspeed; // km/h
	scanf("%d %d", &traveltime, &avgspeed);

	int distance = traveltime * avgspeed;
	double fuel_consumption = 12.0; // km/h
	double min_fuel = distance / fuel_consumption;
	printf("%.3lf\n", min_fuel);
}

File: beginner/1017/FuelSpent.java
// challenge link - https://judge.beecrowd.com/en/problems/view/1017

import java.io.IOException;
import java.util.Scanner;

public class FuelSpent {
	public static void main(String[] args) throws IOException {
		Scanner reader = new Scanner(System.in);
		int tripTimeInHours = reader.nextInt();
		int avgSpeedInKmh = reader.nextInt();
		reader.close();

		int distance = tripTimeInHours * avgSpeedInKmh;
		int fuelEfficiencyInKmh = 12;
		double minFuelReq = distance / (fuelEfficiencyInKmh * 1.0);
		System.out.println(String.format("%.3f", minFuelReq));
	}
}
